modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/info/item/provider/LayoutInfoItemFormProvider.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.layout.admin.web.internal.info.item.provider;
+
+import com.liferay.info.form.InfoForm;
+import com.liferay.info.item.provider.InfoItemFormProvider;
+import com.liferay.layout.admin.web.internal.info.item.LayoutInfoItemFields;
+import com.liferay.portal.kernel.model.Layout;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Adolfo Pérez
+ */
+@Component(service = InfoItemFormProvider.class)
+public class LayoutInfoItemFormProvider
+	implements InfoItemFormProvider<Layout> {
+
+	@Override
+	public InfoForm getInfoForm() {
+		return InfoForm.builder(
+		).infoFieldSetEntry(
+			LayoutInfoItemFields.nameInfoField
+		).infoFieldSetEntry(
+			LayoutInfoItemFields.titleInfoField
+		).infoFieldSetEntry(
+			LayoutInfoItemFields.descriptionInfoField
```

Comment:
The order of the fields are important 
or maybe must be 
Description
Name
Title

If is because the order on layout, ok, but only to be sure.